pkg/antctl/command_runtime.go

```diff
@@ -27,10 +27,13 @@ const (
 var (
 	// runtimeMode tells which mode antctl is running against.
 	runtimeMode string
+	inPod       bool
 )
 
 func init() {
-	if strings.HasPrefix(os.Getenv("POD_NAME"), "antrea-agent") {
+	podName, found := os.LookupEnv("POD_NAME")
```

Comment:
This will make other pods that run antctl will also connect its localhost? Maybe only do it when it's in antrea-agent and antrea-controller

Reply:
Make sense, updated.